src/main/scala/org/tessellation/schema/ΩList.scala

```diff
@@ -0,0 +1,11 @@
+package org.tessellation.schema
+
+abstract class ΩList extends Ω {
+  def ::(elem: Ω): ΩList = new ::(elem, this)
+}
+case class ::(h: Ω, t: ΩList) extends ΩList {
+  override def toString: String = s"$h :: $t"
```

Comment:
instead of `toString` you should use `Show` class

Reply:
I like the idea but in the end, I need to override toString anyway to use such printing syntax everywhere.

Similar pattern as HList uses: https://github.com/milessabin/shapeless/blob/main/core/src/main/scala/shapeless/hlists.scala#L38